### PR TITLE
bullseye rootfs: Add bullseye rootfs image

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Add bullseye rootfs image configuration.

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
- bullseye rootfs tested ok for amd64,armhf,armel,mips64el,mipsel but arm64,i386,mips builds are failing.
	https://bot.staging.kernelci.org/job/rootfs-build-trigger/37/
- Run lava job on bullseye-igt amd64 rootfs image. https://lava.collabora.co.uk/scheduler/job/4489743

**Comment by gctucker** [comment]
@Lakshmipathi  Could you please make this PR specific to the plain `bullseye` build, and have a separate one for `bullseye-igt` once we've checked the plain one works as expected?

File: config/core/rootfs-configs.yaml [hunk 1]
```diff
@@ -13,6 +13,87 @@ rootfs_configs:
...
+      - nosuspend.conf
+      - tar
+      - patch
+      - dir
+      - partx
+      - find
+    script: "scripts/install-bootrr.sh"
+    test_overlay: "overlays/baseline"
+
+  bullseye-igt:
```
Comment (gctucker) [inline config/core/rootfs-configs.yaml:54]: Please split this to a separate commit and make another PR on top of a base one with only the plain `bullseye`.  Then we'll be able to first test just the bullseye update before starting to enable test suites on top of it.

File: config/rootfs/debos/scripts/bullseye-igt.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```
Comment (gctucker) [inline config/rootfs/debos/scripts/bullseye-igt.sh:1]: Same thing, please move this file to a separate commit / PR.

**Review by gctucker — CHANGES_REQUESTED** [review]

File: config/core/rootfs-configs.yaml (hunk 1 above)
Comment (Lakshmipathi) [reply config/core/rootfs-configs.yaml]: Okay removed igt part from this PR. Now it has only `plain bullseye`  rootfs configuration.

**Comment by gctucker** [comment]
@Lakshmipathi Have you tried to build it on staging?

**Comment by Lakshmipathi** [comment]
Job : https://bot.staging.kernelci.org/job/laks/job/rootfs-builder/45/console

**Comment by gctucker** [comment]
The commit message is broken but I'll fix that while merging it...
